.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Description
+
+<!-- Write a brief description of this PR. -->
+
+## Related links
+
+<!-- Write the links related to this PR. Private links should be clearly marked as private, for example, '[FOO COMPANY INTERNAL LINK](https://example.com)'. -->
+
+## Pre-review checklist for the PR author
+
+- [ ] I've confirmed the [self checklist]([https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md](https://docs.google.com/spreadsheets/d/1p5KMcm752iCySHJz0xL1qf6fcLDKprbo/edit#gid=863618628)).
+- [ ] The PR is ready for review.
+
+## In-review checklist for the PR reviewers
+
+The PR reviewers **must** check the checkboxes below before approval.
+
+- [ ] The PR has been properly tested.
+- [ ] The PR has been reviewed.
+
+## Post-review checklist for the PR author
+
+The PR author **must** check the checkboxes below before merging.
+
+- [ ] (Optional) The PR has been properly tested with CARET_report verification.
+- [ ] There are no open discussions or they are tracked via tickets.
```

Comment:
PR is done on a per function basis, so it would be better to check in on the PR conversation instead of tickets.

Reply:
Thank you for your careful review!
Fixed in https://github.com/team-emb4/sched_sim_rust/pull/63/commits/46ff7489943e00ef8eae4382640dd10fed6a7eba.